crate/multiworld-gui/src/main.rs

```diff
@@ -1418,6 +1421,14 @@ impl Application for State {
             Message::ShowLoggingInstructions => if let Err(e) = open("https://github.com/midoshouse/ootr-multiworld/blob/main/assets/doc/logging.md") {
                 return cmd(future::err(e.into()))
             },
+            Message::LaunchProject64 => {
+                let emulator_path = self.pj64_script_path.as_ref().expect("emulator path must be set for Project64 version 3");
+                let pj64_folder_path = Path::new(emulator_path).ancestors().nth(2).unwrap();
+                let pj64_executable_path = pj64_folder_path.join("Project64.exe");
+                if let Err(e) = process::Command::new(&pj64_executable_path).current_dir(pj64_folder_path).spawn() {
```

Comment:
Passing a reference is not necessary here since `pj64_executable_path` is not used again later and `PathBuf` also implements the `AsRef<OsStr>` trait required for the `process::Command::new` parameter.

```suggestion
                if let Err(e) = process::Command::new(pj64_executable_path).current_dir(pj64_folder_path).spawn() {
```

Reply:
Okay yeah that makes sense, so since the variable is not used after that call, its okay to give ownership and let it be release when new() finish?